### ISSUE TITLE
Anti-signal recognition particle (anti-SRP)

### ISSUE THREAD

**Issue by e-lbtn** [obophenotype/human-phenotype-ontology#6800]
**Preferred term label:**
Anti-signal recognition particle (anti-SRP) 

**Synonyms**

**Definition (free text, please give PubMed ID)**
[PMID: 33093664; 29582188; 30639649]

**Parent term (use hpo.jax.org/app)**
Autoimmune antibody positivity(HP:0030057)

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

ORPHA:206569

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
added as HP:0033713, thanks!
